test/meta_test.js

```diff
@@ -40,6 +40,11 @@ suite('Repo Meta Tests', function() {
     assert.equal(taskclusterYml.tasks.$let.node, uiPackageJson.engines.node);
   });
 
+  test('git grep test', async function() {
+    const Taskcluster = await exec('git grep "TaskCluster" ').stdout;
```

Comment:
should there also be a check for "Task Cluster" and "Task cluster"?

Reply:
@jhford should I write a test for "Task Cluster" and "Task cluster"? since the repo name is pretty clear I don't think that it is needed.what you think?

Reply:
I only see one occurrence of that string, but .. yeah, let's test for that too.

Reply:
ok